exercises/06-rock-paper-scissors.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Rock Paper Scissors</title>
+</head>
+<body>
+  <button onclick="
+    userChoice = 'rock';
+    playTheGame();
+    "
+  >Rock</button>
+
+  <button onclick="
+    userChoice = 'paper';
+    playTheGame();
+"
+  >Paper</button>
+
+  <button onclick="
+    userChoice = 'scissors';
+    playTheGame();
+"
+  >Scissors</button>
+
+  <script>
+    const choices = ['rock', 'paper', 'scissors']
+    let computerChoice; 
+    let userChoice;
+    getComputerChoice = () => {
```

Comment:
keep in mind that omitting the `const` identifier is only allowed here because its running on the client side... and a code that is running on the client side, have the window object appended to an unidentified value:

if its not on a client code, it will throw an error that `getComputerChoice` is not defined
example: 
```JS
  localStorage.get() === window.localStorage.get()
```

which means under the hood 
```JS
  getComputerChoice = () => {...} === window.getComputerChoice = () => {...}
```